### PR TITLE
[chrome-api-jstocxxtest] Test methods on already released sCardContext

### PR DESCRIPTION
**Pull request by ViktoriiaKovalova**
This contributes to #947.

### CODE REVIEW

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js [hunk 1]
```diff
@@ -361,6 +363,132 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
...
+                                   'onGetStatusChangeRequested',
+                                   /*requestId=*/ 124, sCardContext,
+                                   /*timeout=*/ {}, [{
+                                     'reader': PNP_NOTIFICATION,
+                                     'currentState': {'unaware': true},
+                                     'currentCount': 0
+                                   }])
+                               .$waitAndVerify();
+      await setSimulatedDevices(
+          [{'id': 123, 'type': SimulationConstants.GEMALTO_DEVICE_TYPE}]);
```
Comment (emaxx-google) [inline smart_card_connector_app/src/chrome-api-jstocxxtest.js:434]: nit: `123` is already used above

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js [hunk 2]
```diff
@@ -361,6 +363,132 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
...
+
+    'tearDown': function() {
+      releasedContext = null;
+    },
+
+    // Test that GetStatusChange requested on an already released context
+    // returns INVALID_HANDLE error.
+    'testGetStatusChange': async function() {
+      expectReportGetStatusChange(/*requestId=*/ 1001, [], 'INVALID_HANDLE');
+      const verifyResult = mockChromeApi
```
Comment (emaxx-google) [inline smart_card_connector_app/src/chrome-api-jstocxxtest.js:453]: nit: Just do `await`, without a variable.

**Review by emaxx-google — APPROVED** [review]

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js (hunk 1 above)
Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js:434]: it's a device id not a request id

Comment (emaxx-google) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js:434]: ack, apologies

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js (hunk 2 above)
Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js]: good catch, thanks!

**Pull request merged**